.travis.yml

```diff
@@ -27,6 +27,7 @@ env:
   global:
     - ANSIBLE_CMD="ansible-playbook -i environments/local -e docker_image_prefix=testing"
     - GRADLE_PROJS_SKIP=""
+    - OW_SCALA_VERSION=2.13
```

Comment:
Great!

Reply:
The scala compilation version was not the culprit.